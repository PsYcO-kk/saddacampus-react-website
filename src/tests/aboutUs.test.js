import {shallow} from 'enzyme';
import React from 'react';
import AboutUs from '../components/AboutUsPage.js';

test("testing for About Us page component" , ()=>{
    const wrapper = shallow(<AboutUs />)
    expect(wrapper).toMatchSnapshot();
})
