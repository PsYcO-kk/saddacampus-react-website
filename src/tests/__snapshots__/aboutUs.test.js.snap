// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for About Us page component 1`] = `<div />`;
